util/telemetry/src/jaeger.rs

```diff
@@ -1,5 +1,6 @@
 use displaydoc::Display;
-use opentelemetry::{sdk, trace::TraceError, KeyValue};
+use opentelemetry::{trace::TraceError, KeyValue};
+use opentelemetry_sdk::{trace, Resource};
```

Comment:
From, https://github.com/open-telemetry/opentelemetry-rust/releases/tag/v0.21.0
> Use the opentelemetry_sdk crate for the SDK types.

Also that crate does use an "_" while the jaeger one uses a "-"